front/js/service.js

```diff
@@ -0,0 +1,183 @@
+/*
+- add axios cdn link in index.html
+     	<scritp src="https://unpkg.com/axios@0.21.1/dist/axios.min.js"></scritp>
+- start scratchy service : 
+    var api = new ScratchyService("http://localhost:5000/api");
+    api;
+- call a function :
+    api.function();|
+*/
+
+
+/**
+ * @constructor
+ * @param {string} apiUrl - Scratchy service API URL, eg: "http://localhost:5000"
+ */
+ function ScratchyService(apiUrl) {
+    let _this = this;
+    this.cacheUsers = []; // List of users, used as cache
+
+//                               ROOM FUNCTIONS
+    /**
+     * @param {string} roomID - room id , eg:60895dd62d1a706830c31f10
+     * @param {string} roomTitle - title of a room , eg:my room
+     * @param {string} roomDescription - description of a room, eg:my room description
+     * @returns {string} - room id
+     */
+    this.updateRoom = async function(roomId,roomTitle,roomDescription){
+        const reponse = await axios.put(apiUrl+'/room/'+roomId, { title: roomTitle , description: roomDescription });
```

Comment:
shoudn't the id be in the response ?

Reply:
yes, the id is in the request. I said that in line 25 : ```  @returns {string} - room id  ```


![screen](https://user-images.githubusercontent.com/48618655/117872410-eb77b580-b29e-11eb-8da3-7d2e067b155a.png)